flexible-classroom/reference/downloads.mdx

```diff
@@ -32,7 +32,7 @@ To demo the full features of the teacher and student across all class types, dow
 </PlatformWrapper>
```

Comment:
What about the iOS download?

Reply:
This is working as it is - [Demo](https://apps.apple.com/us/app/agora-flexible-classroom/id1565281930)